tardis/radiation_field/base.py

```diff
@@ -0,0 +1,62 @@
+import numpy as np
+from astropy import units as u
+
+from tardis.montecarlo.packet_source import BasePacketSource
+from tardis.montecarlo.montecarlo_numba.numba_interface import OpacityState
+
+
+class RadiationField:
```

Comment:
Do we want to be clear about this being a radiationfield*state*?

Reply:
sure